src/main/java/org/spongepowered/common/registry/SpongeGameRegistry.java

```diff
@@ -394,8 +396,14 @@ public Favicon loadFavicon(BufferedImage image) throws IOException {
         return SpongeFavicon.load(image);
     }
 
+    @SuppressWarnings("unchecked")
+    @Override
+    public CraftingRecipeRegistry getCraftingRecipeRegistry() {
+        return (CraftingRecipeRegistry) CraftingManager.getInstance();
+    }
+
     @Override
-    public RecipeRegistry getRecipeRegistry() {
+    public SmeltingRecipeRegistry getSmeltingRecipeRegistry() {
         throw new UnsupportedOperationException(); // TODO
```

Comment:
Not yet. I can either start implementing the smelting recipe registry in this PR, or make another one when this is one is merged.